### DISCUSSION TITLE
A new step in Fief's journey

### DISCUSSION THREAD

**Discussion by frankie567** [fief-dev/fief#342] [Announcements]
The most important change, of course, is the sunsetting of Fief Cloud. This allowed to dramatically simplify the code base and self-hosting deployment.

I'm happy to share today that this work is done and ready to be used 🎉 Starting version 0.28, all Fief instances are now based on a unique workspace.

As promised, we'll work on making self-hosting really easy, with lot of different options to account for every needs, skills and purses. We've already revamped our [self-hosting documentation](https://docs.fief.dev/self-hosting/) and added a new PaaS provider, [Railway](https://docs.fief.dev/self-hosting/deployment/railway/).

# [Read more](https://polar.sh/fief-dev/posts/a-new-step-in-fief-s-journey)

**Comment by pcnoic** [comment]
Good luck on this new leg of the journey @frankie567 and thank you for all the work you put into this project!
